### ISSUE TITLE
Confusion regarding "Passing the tests" for hello world in the cpp track.

### ISSUE THREAD

**Issue by Sjouks** [exercism/cpp#331]
The line:

#if defined(EXERCISM_RUN_ALL_TESTS)

is not included in the test file of hello world. The instructions of the exercise do refer this line, which is confusing to newcomers such as me.

**Comment by sethpeasley** [comment]
I am also very confused by these directions. When I read "get the **first** test compiling, linking and passing" I assume that there are to be multiple tests for this exercise. This assumption is reinforced by the direction to uncomment the **next** test. I looked at the hello_world_test.cpp file, and I see only one test case. I therefore assume my understanding of Exercism and TDD for C++ is incorrect, since you clearly mean there should be more than one test. I ended up punting it by adding it to the end of the test.cpp file, in case it's checked by the Exercism checking engine. 

I suggest you remove this discussion altogether from this exercise's instructions. This exercise seems more about validating that the student has correctly configured all their build infrastructure and can execute CMake, Make, GCC, etc, and test the code. I think it is reasonable to tell the students that this is an introductory example, and then add the discussion of TDD, etc., in Lesson 2 and following.
